src/utils.rs

```diff
@@ -8,10 +8,24 @@ use tilejson::Bounds;
 
 use crate::source::{Query, Xyz};
 
-pub fn prettify_error<E: std::fmt::Display>(message: String) -> impl Fn(E) -> std::io::Error {
-    move |error| std::io::Error::new(std::io::ErrorKind::Other, format!("{message}: {error}"))
+#[macro_export]
+macro_rules! prettify_error {
+    ($error:ident,$info:expr,$($arg:expr),+) => {
+        {
+            std::io::Error::new(std::io::ErrorKind::Other,format!("{}: {}",
+            format!($info,$($arg),*),
+            $error))
```

Comment:
When in a macro, you always want to be explicit with the scope just for hygienic purposes -- I think `::std::...` is always preferable to `std::...` -- keeps things in a global space.